openslides_backend/models/fields.py

```diff
@@ -216,6 +216,8 @@ def get_schema(self) -> Schema:
         schema = self.extend_schema(super().get_schema(), **decimal_schema)
         if not self.required:
             schema["type"] = ["string", "null"]
+        # remove minimum since it is checked in the validate method
+        schema.pop("minimum", None)
```

Comment:
IMO the minimum value is added to be checked!??? Why this should be removed?
Maybe the check has to be suppressed in case of a non required decimal. But not required with minimum value seems curious IMO.

Reply:
As the comment states, the minimum is checked manually in the `validate` method right below that. jsonschema is not capable of handling decimals as string, therefore this has to be done manually and the `minimum` constraint has to be removed from the schema.